src/main.js

```diff
@@ -2,9 +2,8 @@ import WeatherEntity from './weather';
 import style from './style';
 import { handleClick } from './handleClick';
 
-const LitElement = window.LitElement || Object.getPrototypeOf(
-  customElements.get("hui-view"),
-);
+const LitElement = window.LitElement
+  || Object.getPrototypeOf(customElements.get("home-assistant") || customElements.get("hui-view"))
```

Comment:
This won't work on Cast, use `hui-masonry-view` and as fallback `hui-view`

Reply:
Thanks Bram, appreciate it!

Reply:
Another issue with this is, that the `home-assistant` element is not just a LitElement, it has a bunch of mixins you now also get in your card, that you should not get (like themes handling, causing issues in HA 2021.5).

Please fix this asap 😄